pages/api/performance/collect.js

```diff
@@ -0,0 +1,26 @@
+import prisma from '@prisma/index'; // Prisma client
+import { getSession } from 'next-auth/client'; // Session handling
+
+// get performances by specifying the corresponding eventID
+export default async (req, res) => {
+  // Collect session from request
+  const session = await getSession({ req });
+
+  const { eventID } = req.query;
+
+  // If session exists and eventID provided (thus, user is authenticated)
+  // only judges and admins
+  // TODO check they have access to this event
+  if (session && eventID) {
+    // Collect all events from database
+    let performances = await prisma.performance.findMany({
```

Comment:
TODO: We have to add the ability to get performances by event id as well as getting performances by a list of school ids. The former is required for displaying the performances after clicking into the event and the latter is required for feedback sharing. I can take care of that after this gets merged